src/main/java/org/mitre/synthea/helpers/Utilities.java

```diff
@@ -44,7 +44,7 @@ public static long convertTime(String units, long value) {
       case "days":
         return TimeUnit.DAYS.toMillis(value);
       case "years":
-        return TimeUnit.DAYS.toMillis((long) 365.25 * value);
+        return TimeUnit.DAYS.toMillis((long) ((long) 365.25 * value));
```

Comment:
Also, I don't know if the cast or multiplication happens first. If it is the cast, then I believe it will get truncated to 365. As I'm sure you know, this is where some of our leap year pain exists.

Reply:
Cast has higher precedence than multiplication but I agree this looks a mess. Will fix.